extensions/mssql/src/objectManagement/utils.ts

```diff
@@ -42,3 +42,8 @@ export function isValidSQLPassword(password: string, userName: string = 'sa'): b
 	const hasNonAlphas = /\W/.test(password) ? 1 : 0;
 	return !containsUserName && password.length >= 8 && password.length <= 128 && (hasUpperCase + hasLowerCase + hasNumbers + hasNonAlphas >= 3);
 }
+
+// Converts number to two decimal placed string
+export function ConvertNumberToTwoDecimalString(value: number) {
```

Comment:
1. TS functions should be `camelCased` by convention
2. Please add a return type
3. The function name doesn't fully match what it's doing (which also includes adding the MB string)
4. This should be using a localized string `{0} MB` - MB itself should be localized and then not all languages are LTR so we should be using the template for ensuring it's always in the correct order